libs/application/templates/marriage-conditions/src/fields/ApplicationOverview/ApplicationOverview.tsx

```diff
@@ -122,7 +122,7 @@ export const ApplicationOverview: FC<FieldBaseProps> = ({ application }) => {
                 <Text variant="h4">{formatMessage(m.ceremonyDate)}</Text>
                 <Text>
                   {format(
-                    new Date((answers.ceremony as Ceremony).withDate.date),
+                    new Date((answers.ceremony as Ceremony).date),
                     'dd. MMMM, yyyy',
                     { locale: is },
```

Comment:
Isn't date formatting redundant, since we set a locale? Why are we forcing Icelandic locale?

Reply:
im getting the date in english otherwise 🤔